### DISCUSSION TITLE
When can be expect stable version of Prism for MAUI?

### DISCUSSION THREAD

**Discussion by sunilsolanki-alight** [PrismLibrary/Prism#2947] [.NET MAUI]
I would like to thanks @dansiegel for writing this awesome library. It make much easier life of the developer. I am very excited and awaiting for the prism stable version for MAUI. Is there any update by when we will release.

**Comment by dansiegel** [comment]
We are hoping to release an updated .NET 8.0 preview for Prism 9.0 mid September GA being around the GA of .NET 8.0

**Reply by abaslam** [reply]
@dansiegel , Thanks for your all your hard work. As MAUI is now RC1, is there any update regarding this?

**Reply by dansiegel** [reply]
The preview is slightly delayed due to some things we are working on... but we're very close to releasing the preview. I was hoping to get it out this week but it's more likely that it will be another week or two, due to some admin stuff that needs to happen behind the scenes.

**Reply by giuseppenovielli** [reply]
@dansiegel Thanks for feedback!
